build/sources.props

```diff
@@ -19,6 +19,7 @@
       https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json;
       https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json;
       https://dotnet.myget.org/F/aspnetcore-master/api/v3/index.json;
+      https://dotnet.myget.org/F/blazor-dev/api/v3/index.json;
```

Comment:
Out of curiosity, which dependencies did you need from this feed?

Reply:
The things that will continue to build in the Blazor repo (JSInterop and Microsoft.AspNetCore.Blazor.Mono)